Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -1669,8 +1669,7 @@ static ErrorOr<void> scroll_an_element_into_view(DOM::Element& target, Bindings:
 
             // AD-HOC:
             auto* page = document.navigable()->traversable_navigable()->page();
-            auto scale = page->client().device_pixels_per_css_pixel();
-            page->client().page_did_request_scroll_to({ position.left() * scale, position.top() * scale });
+            page->client().page_did_request_scroll_to({ position.left(), position.top() });
```

Comment:
This change breaks navigation to fragment on Acid2. `page_did_request_scroll_to` is supposed to accept values in device pixels and position is stored in css pixels.

Reply:
Any chance you have or can make reduced example for the issue in the inspector?

Reply:
It actually fixes acid2 on my mac, which has a device pixel ratio of 2. Without this patch, we scroll way past the face when clicking "Take The Acid2 Test".

> page_did_request_scroll_to is supposed to accept values in device pixels
It does not, it accepts CSS pixels:
```c++
virtual void page_did_request_scroll_to(CSSPixelPoint) { }
```

Reply:
hmm, I am testing your branch using qt client on retina mac and it breaks scrolling on acid2 for me 🤔  I have a feeling like AppKit and Qt implementations expect different types in `on_scroll_to_point` callback.

Reply:
For reference, this is how `master` behaves on my mac:

https://github.com/SerenityOS/serenity/assets/5600524/a07a746a-305c-41fe-b8c8-73474b1d3e5a

And how this patch behaves:


https://github.com/SerenityOS/serenity/assets/5600524/0ba1bb5f-7bb0-4b17-94b1-5589008a2d07

Reply:
> hmm, I am testing your branch using qt client on retina mac and it breaks scrolling on acid2 for me 🤔 I have a feeling like AppKit and qt implementations expect different types in `on_scroll_to_point` callback.

Ah that could be it, I'll check. Ideally PageHost is the one doing conversions, but not all endpoints comply with that yet

Reply:
Ok after the last push - both Qt and AppKit can take the Acid2 test, the inspector scrolls as expected, and other scroll behavior (i.e. the scroll-by-delta APIs) works well